librz/bin/format/mach0/mach0.c

```diff
@@ -1646,44 +1646,44 @@ static int init_items(struct MACH0_(obj_t) * bin) {
 			is_first_thread = false;
 		} break;
 		case LC_UNIXTHREAD:
-			sdb_set(bin->kv, sdb_fmt("mach0_cmd_%" PFMT64u ".cmd", i), "unixthread", 0);
+			sdb_set(bin->kv, rz_strf(tmpbuf, "mach0_cmd_%" PFMT64u ".cmd", i), "unixthread", 0);
 			if (!is_first_thread) {
-				bprintf("Error: LC_UNIXTHREAD with other threads\n");
+				RZ_LOG_ERROR("mach0: LC_UNIXTHREAD with other threads\n");
 				return false;
 			}
 			// fallthrough
 		case LC_THREAD:
-			sdb_set(bin->kv, sdb_fmt("mach0_cmd_%" PFMT64u ".cmd", i), "thread", 0);
+			sdb_set(bin->kv, rz_strf(tmpbuf, "mach0_cmd_%" PFMT64u ".cmd", i), "thread", 0);
 			if (!parse_thread(bin, &lc, off, is_first_thread)) {
-				bprintf("Cannot parse thread\n");
+				RZ_LOG_ERROR("mach0: Cannot parse thread\n");
 				return false;
 			}
 			is_first_thread = false;
 			break;
 		case LC_LOAD_DYLIB:
 		case LC_LOAD_WEAK_DYLIB:
-			sdb_set(bin->kv, sdb_fmt("mach0_cmd_%" PFMT64u ".cmd", i), "load_dylib", 0);
+			sdb_set(bin->kv, rz_strf(tmpbuf, "mach0_cmd_%" PFMT64u ".cmd", i), "load_dylib", 0);
 			bin->nlibs++;
 			if (!parse_dylib(bin, off)) {
-				bprintf("Cannot parse dylib\n");
+				RZ_LOG_ERROR("mach0: Cannot parse dylib\n");
```

Comment:
i think these `bprintf` are debug only. if they are then these should be `RZ_LOG_DEBUG`